lib/main_containers.pm

```diff
@@ -84,23 +88,44 @@ sub load_host_tests_docker {
     loadtest "containers/container_diff" if (is_opensuse());
 }
 
+sub load_jeos_containers {
```

Comment:
This is wrong. We already have a function loader for podman and docker tests. You are duplicating a lot of code here which is unnecessary.

Reply:
As an urgent task i tried to keep the scheduling clean and isolated. in other case i would have many moving parts. Like many "if else". ofc i would like to avoid and improve this part, which appears twice 

```
if (is_opensuse_jeos()) {
            load_jeos_containers();
        } else
```

but it was the faster approach i came up initially. but it could happen later.
I also think that the *load_jeos_containers* is a good idea for now to not interfere, as jeos is kinda *special* case, and doing this so, we decouple the scheduling from sle.